src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -34,7 +40,15 @@ public Book findByAuthor(@PathVariable String author) throws BookNotFoundExcepti
      * @throws BookNotFoundException (If we can't find a book by this id)
      */
     @PutMapping("/{id}")
-    public Book updateBook(@PathVariable Long id, @RequestBody Book book) throws BookNotFoundException{
+    @ApiOperation(value = "Giving an Id and a Book, it updates that book", response = Book.class)
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Successfuly updated book"),
+        @ApiResponse(code = 404, message = "Book Not Found")
```

Comment:
You can add more errors, for example, that you don't access, error internal server, method not allowed, etc.